app/src/main/java/com/animal_villa/GameRunner.kt

```diff
@@ -5,7 +5,8 @@ import androidx.activity.ComponentActivity
 import com.example.main_animal_villa.R
 
 
-class GameRunner : ComponentActivity(){
+public class GameRunner : ComponentActivity(){
```

Comment:
Specified appropriate visibility modifiers by looking at class diagram and logic https://kotlinlang.org/docs/visibility-modifiers.html